### ISSUE TITLE
The detail of producing bounding box

### ISSUE THREAD

**Issue by DeepRunner** [facebookresearch/VideoPose3D#15]
Hi, Thanks for your code. According to your paper, there are 2 ways to provide bounding box to CPN: one is using Mask R-CNN to extract from original image, the other is ground truth. 
I have 3 question:
1) Is the Mask R-CNN finetuned on Human 3.6M dataset, or directly from Model Zoo which is only trained on COCO?
2) Which backbone is used in Mask R-CNN for extracting bbox, and which for directly produce 2d Key point?
3) Why the Mask R-CNN provided bbox even performs a little better than ground truth? It seems a little confusing.
Thanks a lot!

**Comment by DeepRunner** [comment]
Thank you for your response. 
Is the Mask R-CNN to provide bbox for CPN is also finetuned on H36M?

**Comment by dariopavllo** [comment]
For the experiment "CPN PT" we used the pretrained model, both for 2D keypoints and bounding boxes. For the experiment "CPN FT" we fine-tuned Mask R-CNN on Human3.6M.

**Comment by DeepRunner** [comment]
Thanks!

**Comment by abhikhanna30** [comment]
@dariopavllo Is it possible to provide CPN FT mask rcnn models?

**Comment by abhikhanna30** [comment]
> For the experiment "CPN PT" we used the pretrained model, both for 2D keypoints and bounding boxes. For the experiment "CPN FT" we fine-tuned Mask R-CNN on Human3.6M.

@dariopavllo i am confused by the line "For the experiment "CPN FT" we fine-tuned Mask R-CNN on Human3.6M". Can you tell me how you got the CPN FT mask rcnn models? Or if you can provide it , then that would of great help.

**Comment by dariopavllo** [comment]
Sorry, my previous comment was unclear. This is how it is done:
- D PT: we use the pretrained Detectron (i.e. Mask R-CNN) model.
- D FT: we fine-tune Detectron on Human3.6M.
- CPN PT: we use the pretrained CPN model, and infer using bounding boxes provided by D PT (since in CPN the bounding boxes must be provided externally).
- CPN FT: we fine-tune CPN on Human3.6M, and infer using bounding boxes provided by D FT. In the paper we also have an experiment with ground-truth bounding boxes and the result is similar.

The process is described in detail in Section 5.2 of the paper. Unfortunately, I don't have the model weights on hand right now.
